code/game/machinery/doors/windowdoor.dm

```diff
@@ -65,25 +65,27 @@
 		if(istype(bot))
 			if(density && src.check_access(bot.botcard))
 				open()
-				sleep(50)
+				//sleep(50)
```

Comment:
Turn these sleeps into timers. Windoors can't be bumped like this since they just close immediately.